bbp/default.nix

```diff
@@ -221,6 +221,12 @@ let
         bluepy = callPackage ./nse/bluepy {
         };
 
+        bluepy_0_11_2 = callPackage ./nse/bluepy {
```

Comment:
Simple curiosity : your new bluepy 0.11.2  is API incompatible with the old one that you want to have two separated packages ?